### ISSUE TITLE
PropertyNames is case sensitive

### ISSUE THREAD

**Issue by Jontem** [promaster-sdk/property#25]
PropertyNames in the javascript implementation is case sensitive. This is a problem when serialized and used between the earlier C# implementation which was case insensitive. 

Should we have case sensitive or case insensitive property names?

**Comment by jonaskello** [comment]
From what I recall we decided somewhere back in time that property names should be case insensitive. Perhaps someone else remembers more. @AdamLuotonen do you remember anything about this?

**Comment by AdamLuotonen** [comment]
Yes, I believe we said that they should be case insensitive. However that raises a few other questions how the API should work. For example should GetPropertyNames() return with all lowercase or should casing be kept there? Or should all names be transformed already when parsing PVS/PF? In the end I think having them case sensitive is easier to reason about and would be my choice. I think some applications make use of PascalCase/camelCase and then it is easier to keep their casing for debugging reasons etc.

**Comment by Jontem** [comment]
If we want to preserve the casings the only thing i can come up with is to store original parsed property names at specified key in the pvs object

**Comment by jonaskello** [comment]
Could we not just keep the original property names and just make the string comparison case-insensitive?
